src/main/frontend/src/components/OneReview.js

```diff
@@ -0,0 +1,42 @@
+import React, {useState, useEffect} from 'react';
+
+const OneReview = (props) => {
+    const [chainReview, setChainReview] = useState([]);
+    const [reviewFound, setReviewFound] = useState(true);
+    const reviewId = props.id;
+    const reviewComment = props.comment;
+    const reviewRating = props.rating;
+
+    useEffect(() => {
```

Comment:
We should not have a second `fetch` call inside of this component, as far as I can tell. In `FoodChainShow`, you are obtaining the `foodChain.reviewList`. This list already includes all of the information you need. Fetching it again is unnecessary and bad for performance of your app - you should just use the information passed down via `props` instead!